mappers/milestoneMapper.js

```diff
@@ -0,0 +1,35 @@
+Milestones = require('../models/milestones');
```

Comment:
This is never used, think it passes eslint since you have it as global constant rather than var